pkg/cluster/internal/create/createcontext.go

```diff
@@ -41,6 +41,9 @@ type Context struct {
 	*DerivedConfig
 	Retain      bool         // if we should retain nodes after failing to create.
 	ExecOptions []ExecOption // options to be forwarded to the exec command.
+	//TODO: Refactor this. It is a temporary solution for a phased breakdown of different
```

Comment:
We should probably make the CreateOption be marked temporary as well, particularly since that is actually exported.

Reply:
done